importer/HkEn.py

```diff
@@ -0,0 +1,80 @@
+import dateparser
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class HkEn(Monument):
+
+    def set_heritage_id(self):
+        wlm_code = self.mapping["table_name"].upper()
+        id_no = "{}-{}".format(wlm_code, self.id)
+        self.add_statement("wlm_id", id_no)
+
+    def set_heritage(self):
+        """
+        Set heritage status.
+
+        If declaration_date present and parseable,
+        add it as start_time qualifier
+        """
+        qualifier = None
+        if self.has_non_empty_attribute("declaration_date"):
+            parsed_d = dateparser.parse(self.declaration_date)
+            # will give None when failure
+            if parsed_d:
+                date_dict = utils.datetime_object_to_dict(parsed_d)
+                qualifier = {"start_time": {"time_value": date_dict}}
+
+        heritage = self.mapping["heritage"]["item"]
+        self.add_statement("heritage_status", heritage, qualifier)
+
+    def set_admin_location(self):
+        """Admin location same for all, no subdivisions."""
+        hkong = "Q8646"
```

Comment:
Would it be possible to make use of `region` to add some type of extra to this?

While "Hong Kong Island" (Q19483), "Kowloon" (Q1210635, actually Kowloon Peninsula)  and "New Territories" (Q596660) are just historical subdivisions it's still a guide to figuring out which modern district it corresponds to.

My suggestion is either admin_location with an end_date (1999) or as a qualifier on the hkong value.